### DISCUSSION TITLE
JSON parse error

### DISCUSSION THREAD

**Discussion by samuelpereira** [livewire/livewire#3252] [7. Bugs (archived)] [answered]
On the update of v2.5.2, livewire does not work.

VM207:1 Uncaught SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at new Component (index.js:29)
    at index.js:83
    at Array.forEach (<anonymous>)
    at Livewire.value (index.js:82)
    at HTMLDocument.<anonymous> (home:397)
Component @ index.js:29
(anonymous) @ index.js:83
value @ index.js:82
(anonymous) @ home:397

**Comment by joshhanley** [comment]
@samuelpereira can you share your component class and blade view that is causing the error so we can investigate please?

Thanks!

**Comment by samuelpereira** [comment]
It gives error in every component, I have created a new one for testing and is the same result, works on previous version, on this one no.

Class
namespace App\Http\Livewire;

use Livewire\Component;

class Home extends Component
{
    public function render()
    {
        return view('livewire.home');
    }
}


View
<div wire:poll>
    Current time: {{ now() }}
</div>


Blade Request
@extends('layouts.app')

@section('content')

    @livewire('home')

@endsection

**Comment by joshhanley** [comment]
@samuelpereira sorry I forgot to update you.

For some reason a fix applied in 2.5.1 got reverted by mistake. I have submitted a PR with the fix #3263

Hope this helps!

**Comment by joshhanley** [comment, marked as answer]
@mkwsra a fix has been published. Please upgrade to 2.5.3 🙂

**Comment by samuelpereira** [comment]
Hi,

After the update I have the same error, i saw  #3268, and I have solved the issue!

I have upgraded from Laravel 6.0 and 7.0 to 8.0, and I still using Laravel Ui 3.0.

**Solution:**
I have removed defer attribute from js/app.js on the layouts/app.blade.php and problem is solved!

**On version 2.5.2**
I have downgrade to version 2.5.2 and and looks the same. 
And with version 2.5.2 I also verify that with APP_DEBUG to FALSE livewire don't get any error, but does not work, the poll refresh does not take effect!

So, this update solved all the issues :)

Thanks
